public/javascripts/application.js

```diff
@@ -362,6 +362,22 @@ jQuery.fn.preventDoubleSubmit = function() {
   });
 };
 
+function scrollToErrorIfFound() {
+  var errorField = $j(".LV_invalid_field").first();
+  if (errorField.length !== 0) {
+    $j('html, body').animate({
```

Comment:
Could you use double quotes here and on line 377? (I know this file and many others are super inconsistent about the quotation marks, but we've finally decided we like double. 😆 )